1.hello-world/README.md

```diff
@@ -131,7 +131,7 @@ First, let's POST the message by using Dapr cli in a new command line terminal:
 
 Windows
 ```sh
-dapr send --app-id nodeapp --method neworder --payload '{\"data\": { \"orderId\": \"41\" } }'
+dapr send --app-id nodeapp --method neworder --payload "{\"data\": { \"orderId\": \"41\" } }"
```

Comment:
Please, keep both.

The current one works on PowerShell but not cmd.
The second one works on cmd but not PowerShell.

Reply:
I changed it, see how it looks now.